src/clr-angular/layout/tabs/tabs.ts

```diff
@@ -45,14 +46,17 @@ import { ClrCommonStrings } from '../../utils/i18n/common-strings.interface';
             </ng-container>
         </ul>
         <!--tab content-->
-        <ng-content></ng-content>
+        <ng-container [ngTemplateOutlet]="tabContent.first?.templateRef" ]></ng-container>
     `,
   providers: [IfActiveService, IfOpenService, TabsService, TABS_ID_PROVIDER],
 })
 export class ClrTabs implements AfterContentInit {
   @ContentChildren(ClrTabLink, { descendants: true })
   tabLinkDirectives: QueryList<ClrTabLink>;
 
+  @ContentChildren(ClrTabContent, { descendants: true })
+  tabContent: QueryList<ClrTabContent>;
```

Comment:
If you're always only using the first one, why did you go with `@ContentChildren` instead of `@ContentChild`?

Reply:
Added tests and code for the case without *clrIfActive.
@youdz, the other changes we discussed remain absolutely valid. My suggestion is to finalize this one, so it can allow some immediate vertical style changes and then prioritize the other changes in a separate task, which can get proper prioritization in a normal planning session.

:santa: